plugin/index.js

```diff
@@ -2,6 +2,7 @@
 "use strict";
 
 const _ = require("lodash/fp");
+const { get } = require("lodash");
```

Comment:
Uh oh. 

1. This will break in `node4` which is supported. Can you file a ticket with a 1.0 milestone tag that is basically "CI: ensure code works in supported nodes".
2. For here, just remove and use `_` which is already in scope above.